src/System.CommandLine/tests/System/CommandLine/Tests/ArgumentSyntaxTests.cs

```diff
@@ -617,7 +617,7 @@ public void Option_List_Definition_Error_IllegalName(string name)
             {
                 Parse(string.Empty, syntax =>
                 {
-                    syntax.DefineOptionList(name, Array.Empty<string>());
+                    syntax.DefineOptionList(name, new string[0]);
```

Comment:
Sorry. Because of the size of the PR, I missed the fact that these are tests. I don't think it's worth changing anything here given these are tests.